src/features/user/CompleteSignup/index.tsx

```diff
@@ -501,17 +522,49 @@ export default function CompleteSignup() {
               control={control}
               inputRef={register()}
               defaultValue={true}
-              render={(props: any) => (
-                <ToggleSwitch
-                  checked={props.value}
-                  onChange={(e: any) => props.onChange(e.target.checked)}
-                  inputProps={{ 'aria-label': 'secondary checkbox' }}
-                  id="getNews"
-                />
-              )}
+              render={(props: any) => {
+                return (
+                  <ToggleSwitch
+                    checked={props.value}
+                    onChange={(e: any) => props.onChange(e.target.checked)}
+                    inputProps={{ 'aria-label': 'secondary checkbox' }}
+                    id="getNews"
+                  />
+                );
+              }}
             />
           </div>
 
+          <div className={styles.isPrivateAccountDiv}>
+            <div className={styles.mainText}>
+              <label htmlFor={'terms'} style={{ cursor: 'pointer' }}>
+                {t('editProfile:termAndConditionPart1')}
+                <a
+                  className={styles.termsLink}
+                  href={`https://pp.eco/legal/${i18n.language}/terms`}
+                >
```

Comment:
Maybe this link should get opened in a new tab by adding `rel="noopener noreferrer"` and `target={'_blank'}`?